src/jbuild

```diff
@@ -0,0 +1,6 @@
+(jbuild_version 1)
```

Comment:
quick note: it'd be better to directly write `dune` files (targetting, say, dune 1.1 unless we need something more recent). jbuild only exists for retrocompatibility.